packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -293,7 +294,10 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 
 	const editorPasteText = useCallback(() => {
 		if (editorRef.current) {
-			editorRef.current.replaceSelection(clipboard.readText());
+			void Note.replaceResourceExternalToInternalLinks(clipboard.readText(), { useAbsolutePaths: true })
+				.then((modifiedMd) => {
+					editorRef.current.replaceSelection(modifiedMd);
+				});
```

Comment:
Don't use .then(), use async/await

Reply:
I don't know if my code in [d9d8f60](https://github.com/laurent22/joplin/pull/6865/commits/d9d8f60e2a67f984627fea02a400ea3d35930532) is right. It passed the CI test